CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ elseif(OS_POSIX)
   if(OS_LINUX)
     option(ENABLE_WAYLAND "Enable building with support for Wayland (Linux)" ON)
     option(BUILD_FOR_PPA "Build for PPA distribution" OFF)
+    option(ENABLE_WHATSNEW_LINUX "Enable WhatsNew dialog on Linux" OFF)
```

Comment:
If it on by default and not disable-able on Windows and macOS, I think defaulting it to on is fine and even remove the CMake option.

Reply:
I kept it off becuase enabling it causes a hard-fail if OBS is compiled without MbedTLS. I could perhaps do something like we do for RTMPS which defaults to an "AUTO" option and then only hard fail if it has been manually set to "ON".

Reply:
Forgot about this. Let's make it simple, option with default as off as it is is simple.